glusterd2/commands/volumes/volume-option.go

```diff
@@ -262,6 +265,18 @@ func volumeOptionsHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// Add relevant attributes to the span
+	var optionToSet string
+	for option, value := range req.Options {
+		optionToSet += option + "=" + value + ":"
```

Comment:
question:
is it good to separate options by comma other than colon?

Reply:
Earlier I was just including the option name and so I chose colon so that distinguishing the dots in the option name from the comma separator might be a bit difficult. But since I also include the value now, I guess comma should be fine. I will change it.